torchao/experimental/ops/mps/test/test_lowbit.py

```diff
@@ -4,10 +4,17 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
 
+import os
 import torch
-import torchao_mps_ops
 import unittest
 
+path_libtorchao_ops_mps_aten = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__), "../cmake-out/lib/libtorchao_ops_mps_linear_fp_act_xbit_weight_aten.dylib"
+    )
+)
+torch.ops.load_library(path_libtorchao_ops_mps_aten)
```

Comment:
- If and when we integrate experimental with torchao's pip installation, like pip install . or setup then at that type this lib should be installed in site-packages or some place like that instead of making it available via cmake-out in this way
- Put it under try catch and throw error message if not found